source/opt/local_access_chain_convert_pass.cpp

```diff
@@ -249,8 +249,8 @@ bool LocalAccessChainConvertPass::ConvertLocalAccessChains(ir::Function* func) {
   return modified;
 }
 
-void LocalAccessChainConvertPass::Initialize(ir::Module* module) {
-  InitializeProcessing(module);
+void LocalAccessChainConvertPass::Initialize(ir::IRContext* c) {
+  InitializeProcessing(c);
```

Comment:
Not a change request now, but for future PRs.  Could you move this call to Pass::InitializeProcessing() somewhere that gets called automatically?  I would like to avoid having the passes call this.  Ideally, Pass::InitializeProcessing would be part of the Pass constructor itself.

Reply:
Yes, I want to do something like that.  When I add the code that builds and invalidates things in the IRContext, I will do that.